docs/InitialAwsSetup.md

```diff
@@ -0,0 +1,115 @@
+set it up in the default VPC or create a new VPC just for the bastion server.  The server will need a Public IP Address.  Use the new key pair when prompted.
+
+### Give the bastion server an elastic IP
+In the AWS Console under EC2 - Elastic IP: "Allocate new address".  Assign the new address to your bastion server.  You
+have port 22 opened in your companies firewall to that bastion server.
+
+
+## Create Elastic IP needed for lambda build server
+* Under AWS Console EC2 - Elastic IP -  "Allocate New Address"
+* Choose "Amazon Pool" - Allocate
+* Fill in the name of the Elastic IP as "lambda_build_server"
+
+
+## Setting up the public domain
+
+* Purchase a domain name, externally or through Route53
+* Add it to Route53, so that Route53 manages the DNS records for the domain name
+* Request a wildcard certificate `*.domain.tld`, either externally or using Amazon Certificate Manager (ACM)
+  - Using Domain validation is the easier, and suggested, approach if using ACM. There should be a link after requesting the certificate in ACM.
+  - If you don't want to get a wildcard certificate you need to request `api.domain.tld` and `auth.domain.tld` certificates. If you plan to run multiple Bosslets under the given domain then the certificates will be something _like_ `api.sub.domain.tld` and `auth.sub.domain.tld`, though you can modify the Bosslet configuration `EXTERNAL_FORMAT` value to be whatever you want.
+* Wait until the certificate request has been verificated and the SSL certificate is issued
+  - If you used ACM then you don't need to do anything else
+  - If you requested the certificate externally you need to import the certificate and private key into ACM so that AWS resources can use the SSL certificate
+
+Any Bosslet using the domain can now lookup the certificate(s) needed and attach them to the load balancers to provide HTTPS traffic
+  
+## AWS IAM Permisions
+* Create a Bosslet configuration, using `config/boss_config.py.example` as the template
+  - To verify that the configuration file is correct and includes the needed values you can run `bin/boss-config.py <bosslet.name>`
+* Run `bin/iam_utils.py <bosslet.name> import roles groups policies` to import the initial IAM configuration
+* Remove full permissions from the IAM user and add them to the `XXXXXX` IAM group
+  - Any other Developer or Maintainer should be added to the `XXXXXX` IAM group so they have the needed permissions to manipulate and work with Boss resources
+
+## Create the lambda build server using packer
+
+### Clone the boss-manage.git repository recursively.
+```sh
+$ git clone --recursive https://github.com/jhuapl-boss/boss-manage.git 
+```
+
+Create a virtual environment for working with bossdb and pip install the requirements in it.
+```sh
+$ pip install -r boss-manage.git/requirements.txt
+```
+
+### build Lambda Build Server AMI using Packer
+```sh
+$ bin/packer.py <bosslet_config> lambda --ami-version "" -f
```

Comment:
Should it really be empty quotes for `--ami-version`?

Reply:
Yes, as I don't want it to version the AMI (to be consistent with the current design). If we want we can start versioning the lambda AMI as well. @sandyhider ?